di_website/templates/publications/chapter-nav/nav-item.html

```diff
@@ -18,7 +18,15 @@
             data-current-chapter="1"
         {% endif %}
     >
-      <span class="chapter-title">{{ item.label | title }}:</span>
+      <span class="chapter-title">
+          {% if item.is_foreword %}
+              {{ item.label }}:
+          {% elif item.is_summary %}
+              {{ item.label_type }}:
+          {% else %}
+              {{ item.label }}:
+          {% endif %}
```

Comment:
I don't see where this change has been requested...
For summary pages, this changes the chapter title to `summary`, and that's not what Alice wanted, unless I'm missing something.
https://github.com/devinit/DIwebsite-redesign/issues/774#issuecomment-648244816
Side note, the logic would be the same even without checking for `item.is_foreword`.